src/together/resources/finetune.py

```diff
@@ -100,11 +105,24 @@ def createFinetuneRequest(
     if weight_decay is not None and (weight_decay < 0):
         raise ValueError("Weight decay should be non-negative")
 
+    AVAILABLE_TRAINING_METHODS = {
+        TrainingMethodSFT().method,
+        TrainingMethodDPO().method,
+    }
```

Comment:
Since this is a constant, can you move it to the top of the file (outside of the function and the class definition)?